fac29/NisaInvest-TFB-FE
# Quote component

## Description
This is the quote component

This at the moment is connected to static data in the Carousel folder, but this will be connected to the SUPABASE



## Related Issue
[If applicable, include a link to the related issue]

## How Can This Be Tested?
I have put it on the homepage

so you just need to run 

`npm run dev` and it should be on the front page

## Checklist:
- [ yas b] My code follows the style guidelines of this project
- [ yas b] I have performed a self-review of my own code
- [ yaas b ] My changes generate no new warnings, including `` npm run build ``
